qualang_tools/macros/long_wait.py

```diff
@@ -0,0 +1,23 @@
+from typing import Union
+from qm.qua import declare, wait, for_
+
+MAX_WAIT = 2 ** 31 - 1  # units of clock cycles (4ns)
+
+
+def long_wait(wait_time: Union[float, int], *elements: str):
```

Comment:
Add a note that only Python variables can be used.

And maybe also raise an error if someone tries?

Reply:
Done